cupy/cudnn.pyx

```diff
@@ -1652,7 +1652,8 @@ def convolution_backward_filter(
         zero = <size_t>&float_zero
         one = <size_t>&float_one
 
-    cdef bint use_tensor_core = _should_use_tensor_core(tensor_core, x.dtype)
+    cdef bint use_tensor_core = (
```

Comment:
Please add (TODO?) comment to explain this.

Reply:
Wrote comments.

Reply:
This comment explains exactly what the code already explains.
This is a typical example of useless comment...
Please explain why.

Reply:
Fixed.